scipy/optimize/tests/test_slsqp.py

```diff
@@ -416,18 +459,18 @@ def f1(x):
             return x[0] + x[1] - 2
         def f2(x):
             return x[0] ** 2 - 1
-        sol = minimize(
-            lambda x: x[0]**2 + x[1]**2,
-            x,
-            constraints=({'type':'eq','fun': f1},
-                         {'type':'ineq','fun': f2}),
-            bounds=((0,None), (0,None)),
-            method='SLSQP')
+
+        cons = ({'type': 'eq', 'fun': f1}, {'type': 'ineq', 'fun': f2})
+        bnds = ((0, None), (0, None))
+
+        sol = minimize(lambda x: x[0]**2 + x[1]**2,
+                       x, constraints=cons, bounds=bnds, method='SLSQP')
         x = sol.x
 
         assert_allclose(f1(x), 0, atol=1e-8)
         assert_(f2(x) >= -1e-8)
         assert_(sol.success, sol)
+        _check_kkt(sol, cons, bnds)
```

Comment:
The fact that this is now called in pretty much every test is somewhat comforting.